test/langtools/tools/javac/8264258/MisleadingTypeNotFound.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * @test /nodynamiccopyright/
+ * @bug 8264258
+ * @summary unknown.Class.default gives misleading compilation error
+ * @compile/fail/ref=MisleadingTypeNotFound.out -Xlint:all -Werror -XDrawDiagnostics -XDdev MisleadingTypeNotFound.java
+ */
+package knownpkg;
+
+public class MisleadingTypeNotFound {
+
```

Comment:
It is a bit confusing. The misleading message is not about Type not found, but existing package being reported as non-existent. So we should change these names of classes/files/jbs descriptions etc suitably. 

MisleadingNonExistentPathError ??